Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "TimeZones"
 uuid = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 authors = ["Curtis Vogt <curtis.vogt@gmail.com>"]
-version = "1.5.6"
+version = "1.6.0"
```

Comment:
A bug fix might be more appropriate

Reply:
A minor release is appropriate. This also leaves room for backporting bugfixes